src/trading/makeOrder.se

```diff
@@ -70,15 +70,16 @@ def makeOrder(sender: address, type: int256, attoshares: int256, displayPrice: i
     fxpMoneyEscrowed = escrow[0]
     fxpSharesEscrowed = escrow[1]
 
-    order = [type, market, attoshares, displayPrice, sender, block.number, outcome, fxpMoneyEscrowed, fxpSharesEscrowed, tx.gasprice]
-    orderID = ripemd160(order, items=10)
+    order = [type, market, attoshares, displayPrice, sender, block.timestamp, outcome, fxpMoneyEscrowed, fxpSharesEscrowed]
```

Comment:
Changing this from `block.number` to `block.timestamp` is a pretty significant change.  I'm pretty sure this should be `block.number`.  What was the reason for the change to `block.timestamp`?

Reply:
I did it to match with the removal of the getBlockNumber().  Since you said I should use the timestamp in test_orders.py, I had to use the timestamp in the contracts to get the same hash when calling orders.getOrderIdHash().  I think I had originally tried using contractsFixture.chain.head_state.block_number in test_orders.py, but it wasn't returning the same block number for some reason.

Reply:
`head_state.block_number` should match.  It is possible it is off by one (1 number too low).  This should be pretty easy to test and find out.  When I said you should remove `getBlockNumber()` I just meant that you should be getting the block number from pyethereum, not from the contracts.  My guess for why you were seeing different results is that Pyethereum might be displaying the last mined block, while the contracts will see the pending block (which will be one number higher).